packages/react-core/src/components/Progress/Progress.tsx

```diff
@@ -19,7 +19,7 @@ export interface ProgressProps extends Omit<React.HTMLProps<HTMLDivElement>, 'si
   /** Where the measure percent will be located. */
   measureLocation?: 'outside' | 'inside' | 'top' | 'none';
   /** Status variant of progress. */
-  variant?: 'danger' | 'success' | 'info';
+  variant?: 'danger' | 'success';
```

Comment:
making sure this is intended

Reply:
Yep! `styles.modifiers.info` doesn't exist, so it should be removed from the type.